src/api/Project/mutations/createProject/integration.spec.js

```diff
@@ -0,0 +1,54 @@
+import gql from 'graphql-tag'
+import testClient from '@tests/client'
+import database from '@server/database'
+import resetDatabase from '@tests/resetDatabase'
+
+const mutation = gql`
+  mutation($name: String!, $slug: String!, $description: String, $userId: ID!) {
+    createProject(
+      name: $name
+      slug: $slug
+      description: $description
+      userId: $userId
+    ) {
+      project {
+        id
+        name
+        slug
+        description
+      }
+      errors {
+        field
+        message
+      }
+    }
+  }
+`
+
+describe('The `createProject` mutation', () => {
+  beforeAll(async () => {
+    await resetDatabase()
+    await database.migrate.latest()
+  })
+
+  it('should return the created project on success, with normalized and trimmed values', async done => {
+    await database.seed.run()
+    const projectResponse = await testClient.mutate({
+      variables: {
+        name: 'My awesome project',
+        slug: 'my-awesome-project',
```

Comment:
We should generate the slug automatically using: https://github.com/sindresorhus/slugify